doc/decisions/elektra_web.md

```diff
@@ -0,0 +1,65 @@
+# elektra-web structure
```

Comment:
In [this README.md](https://github.com/ElektraInitiative/libelektra/blob/master/src/tools/web/README.md)? I can add a link once this PR gets merged.